### PR TITLE
Introduce grafink-shell

### PR DESCRIPTION
**Pull request by saucam**
This PR adds a grafink-shell to address requirement #21 of a CLI tool to query the loaded data in JnausGraph (backed by HBase)

### CODE REVIEW

**Comment by saucam** [comment]
This PR is now ready for review @JulienPeloton

File: src/main/scala/com/astrolabsoftware/grafink/CLParser.scala [hunk 1]
```diff
@@ -51,13 +51,15 @@ trait CLParser {
...
-        .required()
+        .optional()
         .action((x, c) => c.copy(startDate = LocalDate.parse(x, dateFormat)))
         .validate(x =>
           if (validateDate(x)) success
           else failure("Option --startdate must be a valid date in format (yyyy-MM-dd)")
         )
-        .text("startdate accepts start day's date for which the job needs to be run (yyyy-MM-dd)")
+        .text(
+          "startdate accepts start day's date for which the job needs to be run (yyyy-MM-dd) defaults to yesterday's date"
```
Comment (JulienPeloton) [inline src/main/scala/com/astrolabsoftware/grafink/CLParser.scala:61]: It would be probably good to mention the default in the README as well

**Review by JulienPeloton — APPROVED** [review]
Looks good to me! I tested the shell with the distant `TestJanusGraph` table without problem. Having the shell backed by Ammonite is definitely a plus - good job!

It can be merged if you do not have more code to push.

File: README.md [hunk 2]
```diff
@@ -299,6 +301,77 @@ for eg:
...
+  e[7ghekg-10cg-6c5-2pz4][47104-similarity->126976],
+  e[5fbzls-1edc-6c5-1rsw][65280-similarity->82688],
+  e[6rzb40-1fk0-6c5-mbk][66816-similarity->28928],
+  e[5fbz7k-1rsw-6c5-1edc][82688-similarity->65280],
+  e[21kfeo-296o-6c5-42kg][105216-similarity->189952],
+  e[7ghe68-2pz4-6c5-10cg][126976-similarity->47104],
+  e[21kf0g-42kg-6c5-296o][189952-similarity->105216]
+)
+
+grafink>g.V("28928").outE("similarity").has("value", 2).asScala.toList
```
Comment (JulienPeloton) [inline README.md:385]: `asScala` needs JavaConverters loaded - could you add `import scala.collection.JavaConverters._` in the example?

File: src/main/scala/com/astrolabsoftware/grafink/CLParser.scala (hunk 1 above)
Comment (saucam) [reply src/main/scala/com/astrolabsoftware/grafink/CLParser.scala:61]: ok

File: README.md (hunk 2 above)
Comment (saucam) [reply README.md:385]: Let me add this to default imports when starting the shell instead

**Comment by saucam** [comment]
Thanks !! Let's merge!!

File: README.md (hunk 2 above)
Comment (JulienPeloton) [reply README.md:385]: sounds good to me!

File: src/main/scala/com/astrolabsoftware/grafink/CLParser.scala (hunk 1 above)
Comment (saucam) [reply src/main/scala/com/astrolabsoftware/grafink/CLParser.scala:61]: updated readme

**Pull request merged**